doc/dom.md

```diff
@@ -167,7 +167,7 @@ While the simdjson library can be used in any project using C++ 11 and above, fi
 ```c++
 padded_string json = R"(  { "foo": 1, "bar": 2 }  )"_padded;
 dom::parser parser;
-dom::object object;
+dom::object object; // invalid until next line
```

Comment:
maybe something along the lines of "invalid until the get() succeeds" - the intended use seems obvious to me, but I'm not sure if anyone might read this literally and try to use it without checking if `error` is not true

Reply:
Agreed